src/Lkl.cc

```diff
@@ -560,7 +565,10 @@ Double_t Lkl::MinimizeLkl(Double_t g,Bool_t gIsFixed,Bool_t isVerbose,Bool_t for
   SetMinuitLink();
 
   // assign value to g and fix it if requested
-  fMinuit->DefineParameter(gGParIndex,fParName[gGParIndex],g+fGShift,fParDelta[gGParIndex],0,0);
+  if(fParName[gGParIndex]=="tau")
```

Comment:
This is not a good thing to do... you are thinking on a  very precise definition of tau with a natural range of uncertainties below 10, but that is by no means compulsory
Why do you want to do this? maybe we can find alternative ways to do it...

Reply:
I saw some "tau=inf" sometimes hence this ugly trick, I agree to discard it for now. Will try to find an example of this problem with the latest fixes, i think the line paper will give me a good opportunity, I'll keep you posted soon (working on it now).